specificatie/verblijfplaats.yaml

```diff
@@ -0,0 +1,324 @@
+      example: "0518200000366054"
+    Postcode:
+      description: |
+        De door PostNL vastgestelde code die bij een bepaalde combinatie van een straatnaam en een huisnummer hoort.
+      type: string
+      pattern: ^[1-9]{1}[0-9]{3}[ ]?[A-Za-z]{2}$
+      example: "2341SX"
+    Straat:
+      type: string
+      pattern: ^[a-zA-Z0-9À-ÿ '\-\']{1,80}$
+      example: "Tulpstraat"
+    VertrokkenOnbekendWaarheen:
+      type: boolean
+      description: |
+        Indicatie dat de persoon is vertrokken naar het buitenland, maar dat niet bekend is waar naar toe.
+    Woonplaats:
+      description: |
+        Een woonplaats is een gedeelte van het grondgebied van de gemeente met een naam.
+      title: woonplaats naam
+      type: string
+      pattern: ^[a-zA-Z0-9À-ÿ '\-\']{1,80}$
+      example: "Duiven"
+    AbstractVerblijfplaatsBeperkt:
+      type: object
+      description: |
+        Gegevens over het verblijf of de woonlocatie van een persoon.
+      required:
+        - type
+      properties:
+        type:
+          type: string
+        functieAdres:
+          $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+        woonplaats:
+          $ref: '#/components/schemas/Woonplaats'
+        adresregel1:
+          $ref: '#/components/schemas/Adresregel1'
+        adresregel2:
+          $ref: '#/components/schemas/Adresregel2'
+        adresregel3:
+          $ref: '#/components/schemas/Adresregel3'
+      discriminator:
+        propertyName: type
+        mapping:
+          GbaVerblijfplaats: '#/components/schemas/GbaVerblijfplaatsBeperkt'
+          Verblijfplaats: '#/components/schemas/VerblijfplaatsBeperkt'
+    VerblijfplaatsBeperkt:
+      allOf:
+        - $ref: '#/components/schemas/AbstractVerblijfplaatsBeperkt'
+        - type: object
+          properties:
+            land:
+              $ref: 'land.yaml#/components/schemas/AbstractLand'
+            vertrokkenOnbekendWaarheen:
+              $ref: '#/components/schemas/VertrokkenOnbekendWaarheen'
+    GbaVerblijfplaatsBeperkt:
+      allOf:
+        - $ref: '#/components/schemas/AbstractVerblijfplaatsBeperkt'
+        - type: object
+          properties:
+            straat:
+              $ref: '#/components/schemas/Straat'
+            huisnummer:
+              $ref: '#/components/schemas/Huisnummer'
+            huisletter:
+              $ref: '#/components/schemas/Huisletter'
+            huisnummertoevoeging:
+              $ref: '#/components/schemas/Huisnummertoevoeging'
+            aanduidingBijHuisnummer:
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+            postcode:
+              $ref: '#/components/schemas/Postcode'
+            locatiebeschrijving:
+              $ref: '#/components/schemas/Locatiebeschrijving'
+            land:
```

Comment:
Hier speelt iets gelijkssoortigs als bij het 'GbaOpschortingBijhouding/datum'. je verwijst hier naar het component 'GbaLand' waarmee je zou verwachten dat je alleen over het GbaLand type zou kunnen beschikken. Binnen Redoc zie ook hier echter weer dat je 3 keuzes krijgt. 

In mijn eigen fork heb ik daarom het component 'GbaLand' aangepast. Ik heb een component 'GbaLandBeperkt' aangemaakt met de volgende vulling:

    GbaLandBeperkt:
      type: string

'GbaLand' heb ik gewijzigd in:

    GbaLand:
      allOf:
      - $ref: '#/components/schemas/AbstractLand'
      - type: object
        properties:
          code:
            $ref: '#/components/schemas/GbaLandBeperkt'

En de $ref in dit attribuut ('GbaVerblijfplaatsBeperkt/land') verwijst nu naar 'GbaLandBeperkt'.

Zie ook hier weer https://melsk-r.github.io/Haal-Centraal-BRP-bevragen/redoc-io#operation/GetPersoon.